github/timestamp_test.go

```diff
@@ -93,6 +93,9 @@ func TestTimstamp_MarshalReflexivity(t *testing.T) {
 		}
 		var got Timestamp
 		err = json.Unmarshal(data, &got)
+		if err != nil {
+			t.Errof("%s: Unmarshal err=%v", tc.desc, err)
```

Comment:
Did you try running this? It looks like this would be a build error. `Errof` should be `Errorf`.